app/controllers/users/sessions_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Users::SessionsController < Devise::SessionsController
+  # Delete web application session and redirect to sso_logout action.
+  def destroy
+    # Delete session
+    if current_user
+      sign_out(current_user)
+      reset_session
+    end
+
+    redirect_to "/users/sessions/sso_logout"
```

Comment:
パスヘルパーを使うとURLの変更に強くなります。


```suggestion
    redirect_to sso_logout_users_sessions_path
```

Reply:
https://github.com/jdkim/llm_meta_server/pull/35/commits/52a1da35e2c8554c0486a0cd2a758bed09c33501 で修正しました。